### ISSUE TITLE
Unexpected folder behaviour

### ISSUE THREAD

**Issue by steven266** [antonreshetov/massCode#61]
## Describe the bug

When I rename a folder and add a slash ("/") into its name the folder is "duplicated".
You will see a folder with the old name and a folder with the new name. If you select one of them both are highlighted. You cannot delete the folder nor rename it (it will result in another folder).

## To Reproduce

- Create a new folder
- Rename the folder and add a slash into its name.

## Expected behavior

The folder should be renamed.

## Screenshots

<img width="191" alt="screenshot" src="https://user-images.githubusercontent.com/5726459/75459575-29313c80-5980-11ea-9bb9-b622194a7c16.png">

## Environment

Version: 1.0.0 (latest available version)
OS version and name: macOS Catalina 10.15.3

**Comment by antonreshetov** [comment]
@steven266 I can't reproduce this bug 
![test](https://user-images.githubusercontent.com/17480987/75470236-6d810480-59a1-11ea-8a60-23609eb414b6.gif)

**Comment by steven266** [comment]
I deleted my storage folder and tried to replicate. Seems like there was an error with the old database files from a previous version. Now it works.
Sorry for the inconveniences.

**Comment by steven266** [comment]
I have deleted the storage folder once more and could replicate it with the current version:

![recording](https://user-images.githubusercontent.com/5726459/75477494-b6828a00-599c-11ea-8939-18e84c1ddee4.gif)

Also the highlighting of selected folders is broken again:

<img width="1112" alt="screen" src="https://user-images.githubusercontent.com/5726459/75477605-dfa31a80-599c-11ea-8dbd-bdece26a9b27.png">

**Comment by antonreshetov** [comment]
Why are you deleting the storage folder at all and still want adequate behavior from the application?

**Comment by steven266** [comment]
I deleted the storage folder and restarted the app to remove the faulty database.
What would you suggest? As far as I see, the app creates a new database on start up, right?

**Comment by antonreshetov** [comment]
That's right, if there is no folder, it will be recreated. But I still can't reproduce this bug.

![Kapture 2020-02-28 at 6 58 51](https://user-images.githubusercontent.com/17480987/75509221-db581b00-59f7-11ea-8779-3aa67dfff14d.gif)
